pyiron_continuum/damask/factory.py

```diff
@@ -0,0 +1,225 @@
+            'discretization':{'t': 10.,'N': 40, 'f_out': 4},
+            'additional': {'f_out': 4}
+        """
+        return DamaskLoading(solver=solver, load_steps=load_steps)
+
+    @staticmethod
+    def material(rotation, elements, phase, homogenization):
+        """
+        creates the damask material
+        Args:
+            rotation(damask.Rotation): damask rotation object
+            elements(str): elements describing the phase
+            phase(dict): a dictionary describing the phase parameters
+            homogenization(dict): a dictionary describing the damask homogenization
+        """
+        return MaterialFactory.config(rotation, elements, phase, homogenization)
+
+    @staticmethod
+    def homogenization(method, parameters):
+        """
+        returns damask homogenization as a dictionary
+        Args:
+            method(str): homogenization method
+            parameters(dict): the required parameters
+        Examples:
+            homogenization(method='SX', parameters={'N_constituents': 1, "mechanical": {"type": "pass"}})
+        """
+        return {method: parameters}
+
+    @staticmethod
+    def phase(composition, lattice, output_list, elasticity, plasticity):
+        """
+        returns a dictionary describing the phases for damask
+        Args:
+            composition(str)
+            lattice(dict)
+            output_list(str)
+            elasticity(dict)
+            plasticity(dict)
+        Examples:
+            phase(composition='Aluminum', lattice= 'cF',
+                  output_list='[F, P, F_e, F_p, L_p, O]',
+                   elasticity=elasticity, plasticity=plasticity)
+            # elasticity= elasticity(type= 'Hooke', C_11= 106.75e9,
+                                        C_12= 60.41e9, C_44=28.34e9)
+            #  plasticity = plasticity(N_sl=[12], a_sl=2.25,
+                                    atol_xi=1.0, dot_gamma_0_sl=0.001,
+                                    h_0_sl_sl=75e6,
+                                    h_sl_sl=[1, 1, 1.4, 1.4, 1.4, 1.4],
+                                    n_sl=20, output=['xi_sl'],
+                                    type='phenopowerlaw', xi_0_sl=[31e6],
+                                    xi_inf_sl=[63e6])
+        """
+        return {composition: {'lattice': lattice,
+                              'mechanical': {'output': output_list,
+                                             'elastic': elasticity,
+                                             'plasticity': plasticity}}}
+
+    @staticmethod
+    def elasticity(**kwargs):
+        """
+        returns a dictionary of elasticity parameters for damask input file
+        Examples:
+             elasticity= elasticity(type= 'Hooke', C_11= 106.75e9,
+                                        C_12= 60.41e9, C_44=28.34e9)
+        """
+        _elast = {}
+        for key, val in kwargs.items():
+            _elast[key] = val
+        return _elast

```

Comment:
```suggestion
return kwargs
```
should be exactly the same if I don't miss anything.

Reply:
I did it for elasticity and plasticity. Thanks!